Tools/scripts/build_options.py

```diff
@@ -81,6 +81,7 @@ def __init__(self,
     Feature('ICE', 'EFI_CURRAWONGECU', 'HAL_EFI_CURRAWONG_ECU_ENABLED', 'Enable EFI Currawong ECU', 0, 'EFI'),
     Feature('ICE', 'EFI_DRONECAN', 'HAL_EFI_DRONECAN_ENABLED', 'Enable EFI DroneCAN', 0, 'EFI'),
     Feature('ICE', 'GENERATOR', 'HAL_GENERATOR_ENABLED', 'Enable Generator', 0, None),
+    Feature('ICE', 'GENERATOR_RICHENPOWER', 'AP_GENERATOR_RICHENPOWER_ENABLED', 'Enable Richenpower Generator', 0, "GENERATOR"),  # noqa
```

Comment:
The comment is a bit funny, "noqa".  I'm not sure we should leave that in.

Reply:
> The comment is a bit funny, "noqa". I'm not sure we should leave that in.

There's a couple of dozen of them in this file.

It's short for "No Quality Assurance".  It's a directive to flake8 that there are known errors on this line and it should just move on with its nit-picking self.  In this case the line is too long.